docs/userguide/configuration.rst

```diff
@@ -596,6 +600,7 @@ Can be one of the following:
 .. _`Elasticsearch`: https://aws.amazon.com/elasticsearch-service/
 .. _`IronCache`: http://www.iron.io/cache
 .. _`CouchDB`: http://www.couchdb.com/
+.. _`CosmosDB`: https://azure.microsoft.com/en-us/services/cosmos-db/
```

Comment:
We can possibly release this with a note that this is still a new / experimental backend so it may not be so well tested in production environments yet. This would make users more cautious and encourage them to perform their own testing.

Reply:
See https://github.com/celery/celery/pull/5185/commits/2e45ac4f03bee7a3f2ddcfd61a3eff8579dcdd34